roles/zuul/meta/main.yml

```diff
@@ -1,2 +1,12 @@
 dependencies:
   - pip
+  - role: logrotate
+    logrotate_configs:
```

Comment:
iirc role defaults' scope only goes as far as the role they live in, and are lowest priority. The caveats to this would be role dependencies and the `include_role` module in 2.2+

http://docs.ansible.com/ansible/playbooks_roles.html#role-default-variables
http://docs.ansible.com/ansible/include_role_module.html

I agree that role defaults should be in `defaults/main.yml`. If they need to be accessed in a different role then `include_role` or role dependencies in `meta` can be exercised. I don't have immense experience with either, so making a recommendation on which one to go with right now is difficult.